releases.yml

```
@@ -20,6 +20,10 @@
       url: /docs/patterns/hero#layouts
       status: New
       notes: We've added new layout options and detailed documentation to the hero pattern.
+    - component: Dense Logo Section
```

Usually I'd write it as "Component" / "Variant" so: "Logo Section / Dense" in this example

```suggestion
    - component: Logo Section / Dense
```